### PR TITLE
Improved commercial request form and smarter design

### PR DESCRIPTION
**Pull request by chriswilk**
# Changes
- Switched to bootstrap
- Add "get started" button to guide visitor to key application
- Add get started blurb to access page
- Add more fields to the commercial key request form. These are the standard questions Syndication ask in the next step.
- Add responsive menu
- Added affixed sub nav to documentation
- Cleaned up documentation
- Added appropriate footer to every page
# Screenshots

Homepage:
image

Get started page:
image

Commercial key form:
image

Sub nav:
image

### CODE REVIEW

File: access/index.html [hunk 1]
```diff
@@ -1,82 +1,178 @@
...
- 	<link type="text/css" rel="stylesheet" href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300,400,400italic">
-	<script src="//code.jquery.com/jquery-1.11.0.min.js"></script>
-	<script type='text/javascript' src='../public/js/lib-knockout-3.1.0.js'></script>
+  	<link href='http://fonts.googleapis.com/css?family=Merriweather' rel='stylesheet' type='text/css'>
+    <link rel="shortcut icon" type="image/png" href="http://assets.guim.co.uk/images/favicons/32x32.250fc306bef36914fafcac2d6cfdef51.ico"></link>
+    <link rel="stylesheet" type="text/css" href="../public/bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="../public/css/lib-normalize.css"></link>
+    <link rel="stylesheet" type="text/css" href="../public/css/common.css"></link>
+ 	  <link rel="stylesheet" type="text/css" href="../public/css/access.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
```
Comment (mchv) [inline access/index.html]: may we keep Jquery CDN instead of the google one?

**Comment by mchv** [comment]
There are several stuffs that are broken and need to be fixed before merging:
- Boolean operators logo are not displayed

screen shot 2015-06-05 at 15 33 20
- Column length are broken  (same screenshot)
- Commons parameters are not displayed

screen shot 2015-06-05 at 15 53 16
- Collapsed menu is only displayed for item documentation page

screen shot 2015-06-05 at 15 34 51
- Front page does not scale for height

screen shot 2015-06-05 at 15 31 44
- There is a typo `Single Items` instead of `Single Item`

screen shot 2015-06-05 at 15 34 26

**Comment by mchv** [comment]
> Added bootstrap for cleaner, responsive design

The site currently use [semantic ui](http://semantic-ui.com/) and is responsive.
`Semantic UI` is [generally considered cleaner with beautifuler default user interface component than bootstrap](http://stackoverflow.com/questions/18886813/semantic-ui-vs-bootstrap) but this is clearly more an opinion than a fact :smiley: 

I am not against switching to bootstrap but let be rational about the reason

**Comment by mchv** [comment]
> Added appropriate footer to every page

Do we really need footer? Even twitter does not have footer.

**Comment by chriswilk** [comment]
@mchv thanks for checking this. I've addressed many of your comments apart from "Front page does not scale for height" so far.

**Comment by chriswilk** [comment]
@mchv all comments have now been addressed. :8ball:

**Comment by mchv** [comment]
Let's merge it then :+1:

**Pull request merged**